src/test/resources/edu/cornell/kfs/krad/service/impl/AccountDelegateGlobalTest.xml

```diff
@@ -1,12 +1,12 @@
 <xmlConversionTestCase>
 
 <oldData>
-<maintainableDocumentContents maintainableImplClass="edu.cornell.kfs.coa.document.CuAccountDelegateGlobalMaintainableImpl"><oldMaintainableObject><edu.cornell.kfs.coa.businessobject.CuAccountDelegateGlobal>
+<maintainableDocumentContents maintainableImplClass="org.kuali.kfs.coa.document.AccountDelegateGlobalMaintainableImpl"><oldMaintainableObject><org.kuali.kfs.coa.businessobject.AccountDelegateGlobal>
```

Comment:
Should we have at least one test case where the pre-conversion side still references the `CuAccountDelegateGlobal` class, to verify that it's being converted to `AccountDelegateGlobal` correctly?